spec/services/deadline_calculator/business_days_spec.rb

```diff
@@ -194,9 +194,11 @@
 
     describe "#time_taken" do
       it "returns the number of business days taken to respond to a case" do
-        closed_case = create(:closed_case)
-        deadline_calculator = described_class.new(closed_case)
-        expect(deadline_calculator.time_taken).to eq 19
+        freeze_time do
```

Comment:
Using Timecop.freeze here seems to cause an rspec `matcher?` error. Using the pre defined `freeze_time` works.